cmd/neofs-adm/docs/subnetwork.md

```diff
@@ -0,0 +1,116 @@
+# Step-by-step NeoFS subnetwork creation
+
+This is a short guide on how to create and manage NeoFS subnetworks.
+This guide considers that side chain and inner ring (alphabet nodes)
+have already been deployed and side chain contains deployed `subnet`
+contract.
+
+## Prerequisites
+
+To follow this guide you need:
+- script hash of `subnet` contract in side chain (little-endian);
+- neo-go side chain RPC endpoint;
+- latest released version of [neofs-adm](https://github.com/nspcc-dev/neofs-node/releases);
+- private key of the NeoFS account;
+- public key of the Storage Node (optional);
+- public keys of the node and client administrators (optional);
+- owner IDs of the NeoFS users (optional).
+
+## Step 1: Subnet creation
+
+```shell
+$ neofs-adm morph subnet create \
+    -r <side_chain_RPC_endpoint> \
+    -a <subnet_script_hash> \
```

Comment:
Is there a reason to define subnetwork address manually? Admin utility can fetch the address from NNS contract.

Reply:
i do not think/know there is a reason for that

Reply:
https://github.com/nspcc-dev/neofs-node/pull/1019